fastlane/Fastfile

```diff
@@ -217,7 +217,7 @@ platform :ios do
   desc 'Updates a release branch for a new beta release'
   lane :new_beta_release do |options|
     ios_betabuild_prechecks(options)
-    ios_update_metadata(options)
+    download_localized_strings_and_metadata(options)
```

Comment:
This is a new lane that does the `.strings` download for WordPress (which Jetpack shares) and the metadata download for WordPress and Jetpack.

I decide to name it like that because I found the release toolkit action name misleading. This name seems more accurate to me. What do you think?

In particular, "download" clearly tells me where the data is coming from, where as "update" could be going both way (e.g. it could be the app updating/uploading metadata to ASC)

Reply:
+ 1000000 on misleading toolkit action names, that has been bothering me for a while and has been a big barrier to entry that made me almost lose my mind the first times I had to work with the toolkit 😅 

In the long term I'm hoping we could rename them all (probably deprecating the existing ones and creating new ones in their place, taking the occasion to modernize and refactor their code), but that's for another day.